scripts/Source/sslAnimationSlots.psc

```diff
@@ -718,6 +718,26 @@ function InvalidateByAnimation(sslBaseAnimation removing)
 	endWhile
 endFunction
 
+function InvalidateByTags(string Tags)
```

Comment:
This function allow invalidate animations on CACHE based on the CACHE Tags and avoid use the ClearAnimCache function.

Was requested by "Sailing Rebel" to avoid the use of the ClearAnimCache when the animation Tags are modified and become very good idea and easy to implement so I made use of the function on the SexLab MCM in the tag relate options. Can be also good idea use it on the Tag related functions in the SexLabFramework script file...